documentation/sphinx/source/release-notes.rst

```diff
@@ -2,6 +2,19 @@
 Release Notes
 #############
 
+6.2.18
+======
+
+Features
+--------
+
+* Add support for setting knobs in fdbcli. `(PR #2773) <https://github.com/apple/foundationdb/pull/2773>`_.
```

Comment:
This seems still requires bouncing the process to use the new knob value.

Will it be clearer if we can add a line about whether or not bouncing the process is required?

Reply:
Maybe the release note is unclear, this allows you to set knobs for the fdbcli binary, not to use the cli to set knobs on other processes.